.travis.yml

```diff
@@ -22,13 +22,13 @@ matrix:
     env: GHCVER='8.0.2' STACK_YAML="$HOME/build/serokell/universum/stack-$GHCVER.yaml"
 
   - ghc: 8.2.2
-    env: GHCVER='8.2.2' STACK_YAML="$HOME/build/serokell/universum/stack.yaml"
+    env: GHCVER='8.2.2' STACK_YAML="$HOME/build/serokell/universum/stack-$GHCVER.yaml"
 
-  - ghc: 8.4.2
-    env: GHCVER='8.4.2' STACK_YAML="$HOME/build/serokell/universum/stack-$GHCVER.yaml"
+  - ghc: 8.4.3
+    env: GHCVER='8.4.3' STACK_YAML="$HOME/build/serokell/universum/stack.yaml"
 
-  - ghc: 8.2.2
-    env: GHCVER='8.2.2' STACK_YAML="$HOME/build/serokell/universum/stack-liquid.yaml"
+  - ghc: 8.4.3
```

Comment:
Why 8.4.3 if the liquid haskell depends on GHC-8.2.2 ?

Reply:
Oops, looks like a mistake, will fix.

Reply:
Fixed. Care to approve?